README.md

```diff
@@ -22,6 +22,12 @@ Kubernetes Helm chart for deploying the complete Resource Optimization Service (
 # Install latest release from GitHub
 ./scripts/install-helm-chart.sh
 
+# Or use local chart for development
```

Comment:
Could it be that there should be a step before installing the helm chart where the `deploy-strimzi.sh` script is called? WDYT?